src/react/actions/__tests__/utils/dispatchActionsList.js

```diff
@@ -157,6 +158,9 @@ export const OPEN_BUCKET_DELETE_DIALOG_ACTION = (bucketName: string): OpenBucket
 export const CLOSE_BUCKET_DELETE_DIALOG_ACTION: CloseBucketDeleteDialogAction =
     { type: 'CLOSE_BUCKET_DELETE_DIALOG' };
 
+export const GET_BUCKET_INFO_SUCCESS_ACTION = (info: BucketInfo): GetBucketInfoSuccess =>
```

Comment:
`GET_BUCKET_INFO_SUCCESS_ACTION` variable can hold "directly" the action object. (No function needed)
I just noticed that we used the function for `OPEN_FOLDER_CREATE_MODAL_ACTION`, `CLOSE_FOLDER_CREATE_MODAL_ACTION`... 
These variables can return their object directly. 
We will change these ones in another PR.

Reply:
done: ok